src/transformState.js

```diff
@@ -6,6 +6,23 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const keys of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
you get single object from there